### PR TITLE
tests: samples: automate testing with PPK for peripheral_power_profiling

### PR DESCRIPTION
**Pull request by nordic-bami**
Add two test cases that will be used in the automatic power tests

### CODE REVIEW

File: samples/bluetooth/peripheral_power_profiling/boards/nrf54l15dk_nrf54l15_cpuapp_auto_conn_advert.overlay [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	aliases {
+		/delete-property/ sw2;
+		/delete-property/ sw3;
+	};
+	/*
+	 * Redefine button0 to use RXD1 - P0.01
```
Comment (maje-emb) [inline samples/bluetooth/peripheral_power_profiling/boards/nrf54l15dk_nrf54l15_cpuapp_auto_conn_advert.overlay:13]:
```suggestion
	 * Redefine button0 to use RXD0 - P0.01
```

File: samples/bluetooth/peripheral_power_profiling/boards/nrf54l15dk_nrf54l15_cpuapp_auto_non_conn_advert.overlay [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	aliases {
+		/delete-property/ sw2;
+		/delete-property/ sw3;
+	};
+	/*
+	 * Redefine button1 to use RXD1 - P0.01
```
Comment (maje-emb) [inline samples/bluetooth/peripheral_power_profiling/boards/nrf54l15dk_nrf54l15_cpuapp_auto_non_conn_advert.overlay:13]:
```suggestion
	 * Redefine button1 to use RXD0 - P0.01
```

File: samples/bluetooth/peripheral_power_profiling/boards/nrf54l15dk_nrf54l15_cpuapp_auto_conn_advert.overlay [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+};
+
+/delete-node/ &button2;
+/delete-node/ &button3;
+
+&uart20 {
+	zephyr,pm-device-runtime-auto;
+};
+
+&uart30 {
```
Comment (maje-emb) [inline samples/bluetooth/peripheral_power_profiling/boards/nrf54l15dk_nrf54l15_cpuapp_auto_conn_advert.overlay:61]: Is it necessary to enable uart30, is it not enough to simply redefine button 0?

Comment (nordic-bami) [reply samples/bluetooth/peripheral_power_profiling/boards/nrf54l15dk_nrf54l15_cpuapp_auto_conn_advert.overlay:61]: Not needed, to be removed

**Review by nordic-pikr — APPROVED** [review]

**Review by nordic-segl — APPROVED** [review]

**Pull request merged**